equistore-core/tests/cpp/CMakeLists.txt

```diff
@@ -63,6 +59,7 @@ foreach(_file_ ${ALL_TESTS})
         # cmake does not find the library on the filesystem and does not add the
         # RPATH to executables linking to it
         BUILD_RPATH ${EQUISTORE_DIR}
+        NO_SYSTEM_FROM_IMPORTED ON
```

Comment:
For what is this needed? I found a description, but I don't understand when this is useful.

> If, for some reason, a consumer should not treat header search paths from imported targets as system search paths, one can set the consumer’s NO_SYSTEM_FROM_IMPORTED target property to true.

Reply:
So we get warnings from these headers when compiling the tests